LayoutTests/TestExpectations

```diff
@@ -7749,3 +7749,6 @@ webkit.org/b/286883 imported/w3c/web-platform-tests/css/CSS2/floats/remove-block
 webkit.org/b/286884 imported/w3c/web-platform-tests/css/CSS2/floats/zero-width-floats-positioning.tentative.html [ ImageOnlyFailure ]
 imported/w3c/web-platform-tests/css/CSS2/floats/float-nowrap-hyphen-rewind-1.html [ ImageOnlyFailure ]
 [ Debug ] imported/w3c/web-platform-tests/css/CSS2/floats/line-pushed-by-floats-crash.html [ Skip ]
+
+# This test is flaky. Resizing the window exposures the failure
+webkit.org/b/288689 svg/text/bidi-text-query.svg [ Failure Pass ImageOnlyFailure ]
```

Comment:
Is there any value to running this test at all? If not, can we just skip it until the bug is fixed?